package/MDAnalysis/coordinates/TRC.py

```diff
@@ -0,0 +1,399 @@
+                                         'FREEFORCE', 'FREEFORCERED',
+                                         'CONSFORCE', 'CONSFORCERED',
+                                         'SHAKEFAILPOSITION',
+                                         'SHAKEFAILPREVPOSITION',
+                                         'LATTICESHIFTS',
+                                         'COSDISPLACEMENTS',
+                                         'STOCHINT', 'NHCVARIABLES',
+                                         'ROTTRANSREFPOS', 'PERTDATA',
+                                         'DISRESEXPAVE',
+                                         'JVALUERESEXPAVE',
+                                         'JVALUERESEPS',
+                                         'JVALUEPERSCALE',
+                                         'ORDERPARAMRESEXPAVE',
+                                         'XRAYRESEXPAVE', 'LEUSBIAS',
+                                         'LEUSBIASBAS', 'ENERGY03',
+                                         'VOLUMEPRESSURE03',
+                                         'FREEENERGYDERIVS03',
+                                         'BFACTOR', 'AEDSS']
+
+        # GROMOS11 trajectories are usually either *.trc or *.trc.gz.
+        # (trj suffix can come up when trajectory obtained from clustering)
+        root, ext = os.path.splitext(self.filename)
+        self.trcfile = util.anyopen(self.filename)
+        if (ext[1:] == "trc") or (ext[1:] == "trj"):
+            self.compression = None
+        else:
+            self.compression = ext[1:]
+
+        # Read and calculate some information about the trajectory
+        self.traj_properties = self._read_traj_properties()
+
+        self._cache = {}
+        self.ts = self._Timestep(self.n_atoms, **self._ts_kwargs)
+
+        self._reopen()
+        self.ts.dt = self.traj_properties["dt"]
+
+        self._read_frame(0)
+
+    @property
+    @cached('n_atoms')
+    def n_atoms(self):
+        """The number of atoms in one frame."""
+        try:
+            return self._read_atom_count()
+        except IOError:
+            return 0
+
+    def _read_atom_count(self):
+        n_atoms = self.traj_properties["n_atoms"]
+        return n_atoms
+
+    @property
+    @cached('n_frames')
+    def n_frames(self):
+        """The number of frames in the trajectory."""
+        try:
+            return self._read_frame_count()
+        except IOError:
+            return 0
+
+    def _read_frame_count(self):
+        n_frames = self.traj_properties["n_frames"]
+        return n_frames
+
+    def _frame_to_ts(self, frameDat, ts):
+        """Convert a frame to a :class:`TimeStep`"""
+        ts.frame = self._frame
+        ts.time = frameDat["time"]
+
+        ts.data['time'] = frameDat["time"]
+        ts.data['step'] = frameDat["step"]
+
+        ts.dimensions = frameDat["dimensions"]
+        ts.positions = frameDat["positions"]
+
+        #
```

Comment:
Just use one comment, no need to break over so many lines.

Reply:
Done